chef/cookbooks/network/recipes/default.rb

```diff
@@ -453,6 +500,22 @@ def kill_nic(nic)
         action :delete
       end
     end
-
+    if node[:platform] == "suse" && node[:platform_version].to_f < 12.0
```

Comment:
@vuntz I am specifically interested in feedback on this part here. Since it will significantly change how we handle SLE11 nodes. Are you ok with this. Or would you prefer the SLE11 code path unchanged. I've the slight feeling that the code in neutron barclamp will get really messy if we try to keep current SLE11 approach  (handling everything in the neutron cookbooks) working as it currently is.